CONTRIBUTING.md

```diff
@@ -1,18 +1,18 @@
 # Contributing to API Elements Documentation
 
 The documentation is built using Sphinx, a Python tool. Assuming you have
-Python 3 installed, the following steps can be used to build the site.
+Python 3 and
+[pipenv](https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv)
+installed, the following steps can be used to build the site.
 
 ```shell
-$ python3 -m venv venv
-$ source venv/bin/activate
-$ pip install -r requirements.txt
+$ pipenv install
 ```
 
 ## Running the Development Server
 
 You can run a local development server to preview changes using the following:
 
 ```shell
-$ make watch
+$ pipenv run make watch
```

Comment:
You can add `[scripts]` section to Pipfile and then ~change this to `pipenv run watch`~ you don't need `make`